app/controllers/invite-user.controller.js

```diff
@@ -60,7 +60,9 @@ async function invite (req, res, next) {
     lodash.set(req, 'session.pageData', { invitee })
     res.redirect(303, formatServicePathsFor(paths.service.teamMembers.invite, externalServiceId))
   } else if (!role) {
-    next(new Error(`Cannot identify role from user input ${roleId}`))
+    req.flash('genericError', 'Select the team member\'s permission level')
```

Comment:
If you use a right single quotation mark (that is, `’` instead of `'`) then it looks nicer and does not need to be escaped.

Reply:
Gotcha! Have updated this as above.